### DISCUSSION TITLE
Compiling a list of resources for Conscious Coliving Opportunities

### DISCUSSION THREAD

**Discussion by nathenf** [life-itself/community#391] [Research]
We would love to have a place on the website to list resources about conscious coliving so that people can have an easily accessible all-in-one place to land and learn about conscious coliving. 

What exactly is conscious coliving vs coliving read our article [What is conscious coliving](https://lifeitself.org/blog/2021/07/14/what-is-conscious-coliving)

Any suggestions/additions welcome.

## Conscious Coliving Directories
- https://diggersanddreamers.org.uk/
- https://www.ic.org/Directory/
- https://ecovillage.org/
- https://wwoof.net/

## Individual Conscious Coliving Projects
- https://lifeitself.org/

## Resources to Learn About Conscious Coliving
- https://dianaleafechristian.org/ 
- https://www.consciouscoliving.com/
- https://www.microsolidarity.cc/

**Comment by top-comengineer** [comment]
Good idea.
As a senior web developer, I can help so you can build your idea.
By now, I have built so many website that can help humen' s daliy life.
These are getting good reputation from so many people.

**Comment by rufuspollock** [comment]
@nathenf does this just cover sites that are **lists** of coliving opportunities or individual coliving projects?

Also is it coliving opportunities in general or more **conscious** coliving opportunities (e.g. intentional communities, ecovillages etc)?

BTW: how does this discussion relate to the existing https://github.com/orgs/life-itself/discussions/144? (I'm guessing we kind of want to merge these or at least flag the other discussion in each thread)

**Reply by nathenf** [reply]
Good question, thanks for helping me clarify. I think we are focussed mostly on conscious coliving as a filter, I can link to your article on conscious coliving for helping with this definition. 

Updating accordingly

**Comment by top-comengineer** [comment]
I don't care of your general life style.
I am only intersted in development.
But, I want to let you know that if you offer good idea to build into practise, I will do my best for that and produce good result.

**Comment by mattosborn** [comment]
Very interested in this!

 I’m especially interested in the idea of mapping co-living and community projects with a focus on how they relate to different sub-cultures, value systems, ideologies, etc, and especially those that have been emerging in recent years. I love Nadia Asparouhova's work on this type of ["cartography"](https://www.youtube.com/watch?v=jvAyJImbEeA) where she maps out the different [tribes of climate](https://nadia.xyz/climate-tribes), and also Roote Foundation’s [Wisdom Age piece](https://www.roote.co/wisdom-age) that looks at the different tribes of transition. I wonder if a similar thing could be done for the different "tribes of community living”.

I think there’s a ton of value in trying to facilitate the networking of something that currently feels very fragmented. One of the fundamental shared values behind many of these different tribes is the practice of "coming together" – a rediscovery of community and a pushback against individualism. So it seems only natural to use the digital tools we have available to facilitate this more broadly, to connect communities with each other, to orient and connect newcomers to communities that share their value systems, to connect prospective community builders with existing communities that they can learn from, to connect curious minds with learning resources, and so on.

**Reply by rufuspollock** [reply]
@mattosborn great to hear of your interest. In terms of the mapping, did you see the visualizations / maps on https://lifeitself.org/ecosystem (see the blog post https://lifeitself.org/blog/2022/10/25/ecosystem-mapping-update)

> I think there’s a ton of value in trying to facilitate the networking of something that currently feels very fragmented.

That would be great ... though to start with just having one simple list would be great 😄

**Reply by mattosborn** [reply]
> @mattosborn great to hear of your interest. In terms of the mapping, did you see the visualizations / maps on https://lifeitself.org/ecosystem (see the blog post https://lifeitself.org/blog/2022/10/25/ecosystem-mapping-update)

Ya I saw the ecosystem page & love it the direction. 

> That would be great ... though to start with just having one simple list would be great 😄

Agree! And to be clear, all I really meant there by "networking" is to provide self-service resources (for example, a list of projects) that can help to connect people, orgs, communities together. The layers on top of this – such as the orientation triangle offered on the ecosystem mapping page – just make this process more efficient.

I did some scraping a few months ago to compile a list of projects across the world that I can share. It was really just a starting point and only contains a small fraction of projects that exist, but it still contains about 200-300 projects. There are still a few directories that I haven't yet scraped too.

Right now it only contains only name + URL. From this I was planning to attach the project's geolocation (lat/long and place name data). I have a script that fetches this from google maps API but haven't yet run this across the whole dataset. This is a little tricky too because not all projects exist in a single location, or even have a physical location at all, though it should work fine for the majority.

Then there's a bunch of other metadata that would be interesting and useful. I was playing around with using AI (of course) to augment  this dataset with objective data (e.g. year founded, co-founders) and subjective data (e.g. categorisations). 

>  I think we are focussed mostly on conscious coliving as a filter, I can link to your article on conscious coliving for helping with this definition.

I think "conscious co-living" as a filter is really hard to apply because there isn't really a good measure of it. It's subjective and non-binary. And if that measure has to be done manually then you have to start with a set of all projects to begin with anyway, which is why I ended up leaning towards the idea of casting a wide net and then adding categorisations to it using different lenses.

**Reply by rufuspollock** [reply]
> I did some scraping a few months ago to compile a list of projects across the world that I can share. It was really just a starting point and only contains a small fraction of projects that exist, but it still contains about 200-300 projects. There are still a few directories that I haven't yet scraped too.

Fantastic, can you share that e.g. in a google spreadsheet or similar?

BTW i have a list i made a few years ago with about a 100 and can share that too.

> I think "conscious co-living" as a filter is really hard to apply because there isn't really a good measure of it. It's subjective and non-binary. And if that measure has to be done manually then you have to start with a set of all projects to begin with anyway, which is why I ended up leaning towards the idea of casting a wide net and then adding categorisations to it using different lenses.

For now we can just have a list and then filter later.

**Comment by nathenf** [comment]
Hey @mattosborn awesome to hear from you and definitely keen to get this moving along!

**Comment by mattosborn** [comment]
[Here's a start](https://docs.google.com/spreadsheets/d/1FlLWdW5k4PKJ1T6rdcXy2cSwWUO-MInxCfpkCtuWhVY/edit#gid=2101869082)

Sheets:
- **List of lists** – Directories and lists to scrape from
- **Assorted** – Communities from a couple of different sources. Two of the sources are listed in the "List of lists" sheet, but I can't remember the other sources. This also has some initial (but incomplete) location data.
- **ic.org** – Initial scrape of the ic.org directory. Probably also need to scrape each ic.org community page to at least get the official URLs.

I've set this sheet to public read-only so feel free to request edit access. & I'll carry on scraping the unscraped directories as I get time to.

**Reply by rufuspollock** [reply]
@mattosborn amazing 🎉🙏

Question: would you be up for moving dev work to e.g. https://github.com/life-itself/intentional-communities? We could start by opening an issue there around stuff like scraping ic.org, structure of db, job stories etc.

PS: i should have flagged earlier in this thread this existing discussion: https://github.com/orgs/life-itself/discussions/144 (this covered scraping ic.org - we pinged them to ask for the DB and never got a clear answer)

**Reply by Liyubov** [reply]
Just to reply to this on **ic.org** communities research and co-research with them, here is the depersonalised answer on this from Co-Director. Ads, Development and Help Desk. Foundation for Intentional Community (FIC) :

> I personally and very interested in exploring ways to visualize parts of our dataset. I understand that you would like an export. I imagine that my colleagues might also be interested in hearing what you have in mind to do with as well as partnering with you about what we might be interested in the outcome being.  To answer your question about common discussion space. If I am understanding that correctly, I’ll mention [forum.ic.org](http://forum.ic.org/).

We can reach out to them again once we indeed identify the clear direction of this research. 

In reply to Rufus and @mattosborn

**Comment by karlsteyaert** [comment]
This is an extensive list of intentional community projects that someone I know compiled: https://tuckerwalsh.medium.com/transformational-communities-cd9e41053423

**Reply by rufuspollock** [reply]
@karlsteyaert really useful! thank-you! 🙏

**Comment by Postnormalval** [comment]
I found that also https://communityfinders.com/community-directories/

**Reply by rufuspollock** [reply]
Really great link 🙏

![image](https://user-images.githubusercontent.com/180658/236636740-5fe1823f-b410-45c0-b06b-d754edb6be43.png)

**Reply by asimong** [reply]
I have talked with Cynthia and intend to remain in touch.

**Comment by laurenwigmore** [comment]
mentions in whatsapp chat:
-  Santiniketan, India, which itself started as a conscious coliving community in 1901
- https://www.traditionaldreamfactory.com/ (with note there is nowhere to go there)
- https://labs.thegarden.pt/ - with note feedback one person had an amazing time and they are in an interesting period ("They have a lot of infrastructure that is conducive to community and embodied rituals, and are at various inflection points for firming up their social rhythms and community continuity. It would be an interesting to time to join in on the project")

**Comment by Charlotte-br560** [comment]
Your effort to create a centralized hub for conscious coliving information is commendable.
